### PR TITLE
Add single-node tutorial

### PR DESCRIPTION
**Pull request by pmatulis**
Leave this new tutorial hidden in the navigation for now. After the addition of another tutorial in a second PR the old tutorial material will be unpublished in a third PR.

### CODE REVIEW

**Comment by pmatulis** [comment]
@utkarsh2102 Can you advise on the failed CI test?

File: docs/tutorial-2/single-node.rst [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+Add the three disks to the cluster by using the :command:`disk add` command:
+
+.. code-block:: none
+
+   sudo microceph disk add /dev/sda --wipe
+   sudo microceph disk add /dev/sdb --wipe
+   sudo microceph disk add /dev/sdc --wipe
+
+Rechcek status:
```
Comment (sabaini) [inline docs/tutorial-2/single-node.rst:100]: Typo: "Recheck"

File: docs/tutorial-2/single-node.rst [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+.. code-block:: none
+
+   sudo microceph status
+
+The output should now show three disks and the additional presense of the OSD
```
Comment (sabaini) [inline docs/tutorial-2/single-node.rst:106]: Typo: "presence"

File: docs/tutorial-2/single-node.rst [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+   sda           8:0    0 931.5G  0 disk
+   sdb           8:16   0 931.5G  0 disk
+   sdc           8:32   0 931.5G  0 disk
+   sdd           8:48   0 931.5G  0 disk
+   nvme0n1     259:0    0 372.6G  0 disk
+   ├─nvme0n1p1 259:1    0   512M  0 part /boot/efi
+   └─nvme0n1p2 259:2    0 372.1G  0 part /
+
+There are four disks available, here we will use disks ``/dev/sda``,
+``/dev/sdb``, and ``/dev/sdc``.
```
Comment (sabaini) [inline docs/tutorial-2/single-node.rst:37]: Not sure if it goes without saying, but might want to add a caution that we're going to wipe the data? Either here or below when actually adding the disks

File: docs/tutorial-2/single-node.rst [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+The output should now show three disks and the additional presense of the OSD
+service:
+
+.. code-block:: none
+
+   MicroCeph deployment summary:
+   - node-mees (10.246.114.49)
+       Services: mds, mgr, mon, osd
+         Disks: 3
+
```
Comment (sabaini) [inline docs/tutorial-2/single-node.rst:115]: I wonder if we may want to show the output of `ceph -s` as well to demonstrate we now have a full, healthy ceph cluster

**Review by sabaini — CHANGES_REQUESTED** [review]
Hi, left some inline commentary. Thanks!

**Review by sabaini — APPROVED** [review]
lgtm, cheers

**Pull request merged**